### DISCUSSION TITLE
Manage albums and sub-albums...

### DISCUSSION THREAD

**Discussion by HeartBtz** [chevereto/chevereto#65] [Ideas]
Hello, 

As I try to manipulate my albums, I realize that it is quite complicated to move between albums and sub-albums... What do you think about creating a sort of tree structure that would make it easier to navigate through the parent albums (like a file system?)?

**Comment by rodber** [comment]
Hi @HeartBtz, thanks for your post.

I do agree on that this need more attention! I've raised an issue.

https://github.com/chevereto/chevereto/issues/73

**Comment by HeartBtz** [comment]
Hello, 

Thank you for your attention to my comment. 
I can't wait to see the next Chevereto release!
